collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/ProgressDialogFragment.java

```diff
@@ -104,6 +108,10 @@ private void setupView(@NonNull AlertDialog dialog) {
             ((TextView) dialogView.findViewById(R.id.message)).setText(getArguments().getString(MESSAGE));
         }
 
+        if (getArguments() != null && Objects.equals(getArguments().getBoolean(CANCELABLE), false)) {
```

Comment:
I think this could just be:

```
if (getArguments() != null) {
    dialog.setCancelable(getArguments().getBoolean(CANCELABLE), true));
}
```
Is that right?